doc-content/enterprise-only/migration/migration-tool-niogit-7.4-proc.adoc

```diff
@@ -0,0 +1,217 @@
+. In the extracted `{PRODUCT_FILE}-add-ons` folder, extract the `{PRODUCT_INIT}-7.5-migration-tool.zip` sub-folder. The migration tool is in the `bin` directory.
+. In your command terminal, navigate to the temporary directory where you extracted the `{PRODUCT_INIT}-7.5-migration-tool` folder and run the migration tool. The `${PRODUCT_INIT_CAP}_NIOGIT_DIR` portion is the path to the `.niogit` directory that you previously copied to the {PRODUCT} 7.5 installation.
++
+--
+On Linux or UNIX-based systems:
+[source,subs="attributes+"]
+----
+$ cd $INSTALL_DIR/{PRODUCT_INIT}-7.5-migration-tool/bin
+$ ./migration-tool.sh -t ${PRODUCT_INIT_CAP}_NIOGIT_DIR
+----
+
+On Windows:
+[source,subs="attributes+"]
+ifdef::DM[]
+----
+$ cd $INSTALL_DIR\rhdm-7.5-migration-tool/bin
+$ migration-tool.bat -t ${PRODUCT_INIT_CAP}_NIOGIT_DIR
+----
+endif::DM[]
+ifdef::PAM[]
+----
+$ cd $INSTALL_DIR\rhpam-7.5-migration-tool\bin
+$ migration-tool.bat -t ${PRODUCT_INIT_CAP}_NIOGIT_DIR
+----
+endif::PAM[]
+
+In the command prompt that appears, the following options are displayed:
+
+* *Project structure migration*: Migrates the {PRODUCT} 7.4 project repository structure to the new project-oriented structure used in {PRODUCT} {PRODUCT_VERSION}.
+* *System configuration directory structure migration*: Migrates the `system.git` repository structure used in {PRODUCT} 7.4 to the new structure used in {PRODUCT} 7.5. This migration option requires the project structure migration to be executed first.
+ifdef::PAM[]
+* *Forms migration*: Migrates forms created in the {PRODUCT_OLD} forms designer to the new forms designer. This migration option requires the project structure migration and system configuration directory structure migration to be executed first.
+endif::PAM[]
+* *POMs migration:* Updates `pom.xml` files with dependencies required for {PRODUCT} 7.5. This migration option requires the
+ifdef::PAM[]
+project structure migration, system configuration directory structure migration, and forms migration
+endif::PAM[]
+ifdef::DM[]
+project structure migration and system configuration directory structure migration
+endif::DM[]
+to be executed first.
+* *All:* Runs all migration options in sequence.
+* *Exit:* Exits the migration tool.
+--
+. Select the option to run *All* migrations in sequence.
++
+NOTE: If you prefer to run one migration option at a time, select and run the first individual migration option. After the tool runs, re-run the {CENTRAL} migration tool and select the next individual migration option in the sequence.
++
+
+. Enter `yes` each time you are prompted to run a specific migration option.
++
+For the POMs migration option, if you want to include a path to an external `pom-migration.json` file that you created previously, enter `yes` when prompted and enter the path.
++
+. After the tool finishes running, enter the option to *Exit* the migration tool.
++
```

Comment:
Should not this and the next one be under "Results" and not a step?

Reply:
I don't think so. They are both actions.